st2client/st2client/commands/resource.py

```diff
@@ -154,6 +155,7 @@ def get_resource_by_pk(self, pk, **kwargs):
         try:
             instance = self.manager.get_by_id(pk, **kwargs)
         except Exception as e:
+            traceback.print_exc()
```

Comment:
It's intentional. Dude, we don't print the exception and bubble up a weird exception. This is not just a --debug case. This is useful.